dipy/align/tests/test_imaffine.py

```diff
@@ -172,7 +172,7 @@ def test_align_origins_3d():
 
 def test_affreg_all_transforms():
     # Test affine registration using all transforms with typical settings
-    for ttype in factors.keys():
+    for ttype in sorted(factors):
```

Comment:
Comments added. Actually only one of the tests needed the sort, but I think it is ok to still sort the keys in the other three places, just in case we extend the tests in the future.